lib/screens/login_screen.dart

```diff
@@ -64,11 +63,19 @@ class LoginScreenState extends State<LoginScreen> {
       });
       if (_rememberMe) {
         setNamiPassword(_password);
+      } else {
+        deleteNamiPassword();
       }
       setNamiLoginId(_mitgliedsnummer);
-      WidgetsBinding.instance.addPostFrameCallback((_) {
-        AppStateHandler().setLoadDataState(context, loadAll: true);
-      });
+      if (AppStateHandler().currentState == AppState.loggedOut) {
+        WidgetsBinding.instance.addPostFrameCallback((_) {
+          AppStateHandler().setLoadDataState(loadAll: true);
+        });
+      } else if (AppStateHandler().currentState == AppState.relogin) {
```

Comment:
Angenommen ich lande im relogin und melde mich mit anderen Daten ein. Dann müssen die Daten gelöscht und neu geladen werden

Reply:
Oh gut beobachtet. Fixed in [809021e](https://github.com/digital-scouts/dpsg-nami-app/pull/66/commits/809021ebf078f37a3ac087e5ed3d2fd515adeec7)